src/views/Shapes/Capsule.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Modifiers } from "../../utils/modifiers";
+import { Rectangle } from "./Rectangle";
+
+type CapsuleProps = Omit<Modifiers,"backgroundColor"> & {
+    fill: string
+}
+
+export const Capsule: React.FC<CapsuleProps> = ({
+    fill,
+    opacity,
+    frame,
+    scaleEffect,
+    padding,
+    border,
+    shadow,
+    zIndex,
+    style,
+    onAppear,
+    onDisappear,
+}) => {
+
+    const width = frame?.width || "100%";
```

Comment:
instead of this, you can add
`  frame = { width: "100%", height: "100%" }, `
when you destructure the props in the fn component definition

Reply:
Fixed in [8c172e0](https://github.com/andrew-levy/swiftui-react-native/pull/10/commits/8c172e0af1b8b252edf1857088c13ab4c7a7fe59)